wordvecspace/tornado_wvs.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import tornado.ioloop
+import tornado.web
+
+from kwikapi.tornado import RequestHandler
+from kwikapi import API
+from logging import Logger
+
+from typing import Union
```

Comment:
@RamanjaneyuluIdavalapati This is a python standard module. Please arrange modules like we discussed.

Reply:
@prashanthellina done